src/components/Nav.jsx

```diff
@@ -1,11 +1,25 @@
 import {Link} from 'react-router-dom';
+import Header from './Header';
+import {FormControlLabel, Switch} from '@mui/material';
+
 import "./nav.css"
-function Nav() {
+function Nav({toggleTheme, setToggleTheme}) {    
+    const toggleHandle = () => {
+        if(toggleTheme) {
+            setToggleTheme(false)
+        } else {
+            setToggleTheme(true)
+        }
+    } 
     return (
-        <ul className='nav'>
-            <li><Link to="/">Home</Link></li>
-            <li><Link>Topics</Link></li>
-            <li><Link>Topics</Link></li>
+        <ul className={`nav ${toggleTheme? "nav--light": ""}`}>
+            <div className={`nav-left ${toggleTheme? "nav-left--light": ""}`}>
+                <h1>nc-news</h1>
+                <Link to="/">Home</Link>
+                <Link>Articles</Link>
+                <Link>Topics</Link>
+            </div>
+            <FormControlLabel control={<Switch color="primary" onChange={toggleHandle}/>} label="light-mode"></FormControlLabel>
```

Comment:
Currently, it looks like your toggle is displaying the label of 'light-mode' regardless of whether the user is in dark or light mode